File: Project-Euler/multiplesOf3And5.c
#include <stdio.h>

int multiplesOf3And5(int number) {
    int sum = 0;

    for (int x = 0; x < number; x++) {
        if (x % 3 == 0 || x % 5 == 0) {
            sum += x;
        }
    }

    return sum;
}

int main() {
    printf("%d\n", multiplesOf3And5((1000)));
    return 0;
}

File: Project-Euler/multiples_of_3_and_5.py
def multiples_of_3_and_5(number):
  sum = 0

  for x in range(number):
    if (x % 3 == 0 or x % 5 == 0):
      sum += x
  
  return sum;

print(multiples_of_3_and_5(1000))